ChangeLog.md

```diff
@@ -1,5 +1,18 @@
 ## 2016.02.02 version 1.1.1
 * Azure RemoteApp: Organizational Unit in Azure RemoteApp RDFE cmdlets now accepts Unicode characters.
+* Azure LogicApp: New cmdlets for managing LogicApps  
+  * Get-AzureLogicAppAccessKey
+  * Get-AzureLogicApp
+  * Get-AzureLogicAppRunAction
+  * Get-AzureLogicAppRunHistory
+  * Get-AzureLogicAppTrigger
+  * Get-AzureLogicAppTriggerHistory
+  * New-AzureLogicApp
+  * Remove-AzureLogicApp
+  * Start-AzureLogicApp
+  * Set-AzureLogicAppAccessKey
+  * Set-AzureLogicApp
+  * Stop-AzureLogicAppRun
```

Comment:
[Fixed]: Moved to new section as suggested. Below tag indicates next release in March. I just want to know, What is the process if we want to release these changes early?

From: Hovsep [mailto:notifications@github.com]
Sent: Monday, February 1, 2016 3:22 PM
To: Azure/azure-powershell <azure-powershell@noreply.github.com>
Cc: Pankaj Singh Negi <Pankaj.Negi@microsoft.com>
Subject: Re: [azure-powershell] Azure Logic App PowerShell Cmdlets (#1753)


In ChangeLog.md<https://github.com/Azure/azure-powershell/pull/1753#discussion_r51498279>:

> @@ -1,5 +1,18 @@

>  ## 2016.02.02 version 1.1.1

>  * Azure RemoteApp: Organizational Unit in Azure RemoteApp RDFE cmdlets now accepts Unicode characters.

> +* Azure LogicApp: New cmdlets for managing LogicApps

> +  * Get-AzureLogicAppAccessKey

> +  * Get-AzureLogicApp

> +  * Get-AzureLogicAppRunAction

> +  * Get-AzureLogicAppRunHistory

> +  * Get-AzureLogicAppTrigger

> +  * Get-AzureLogicAppTriggerHistory

> +  * New-AzureLogicApp

> +  * Remove-AzureLogicApp

> +  * Start-AzureLogicApp

> +  * Set-AzureLogicAppAccessKey

> +  * Set-AzureLogicApp

> +  * Stop-AzureLogicAppRun

Your changes will not go out as part of February release. Please add a new tag

2016.03.08 version 1.3.0

and then add your change log text.

—
Reply to this email directly or view it on GitHub<https://github.com/Azure/azure-powershell/pull/1753/files#r51498279>.